### PR TITLE
[TCBZ2419] Add PcdProgressBarBackgroundColor to hold the background color of progress bar.

### PR DESCRIPTION
**Pull request by wenbhou**
Add PcdProgressBarBackgroundColor to hold the background color of progress bar, so that it can be changed in platform config.

### CODE REVIEW

**Comment by corthon** [comment]
Could you please add "MU_CHANGE" comments around all these changes? Thanks!

File: MdeModulePkg/MdeModulePkg.dec [hunk 1]
```diff
@@ -1593,6 +1593,10 @@
   # @Prompt Enable Capsule On Disk support.
   gEfiMdeModulePkgTokenSpaceGuid.PcdCapsuleOnDiskSupport|FALSE|BOOLEAN|0x0000002d
 
+  ## This PCD holds the background color of progress bar.
+  # @Prompt Describes background color of progress bar.
```
Comment (corthon) [inline MdeModulePkg/MdeModulePkg.dec:1603]: Can you add a comment about byte order (i.e. Reserved, Red, Green, Blue)?

**Review by corthon — CHANGES_REQUESTED** [review]

File: MdeModulePkg/MdeModulePkg.dec (hunk 1 above)
Comment (wenbhou) [reply MdeModulePkg/MdeModulePkg.dec:1603]: Done.

**Comment by wenbhou** [comment]
> ld you please add "MU_CHANGE" comments around all these changes? Thanks!

Done.

**Comment by corthon** [comment]
Linked to TianoCore bug:
https://bugzilla.tianocore.org/show_bug.cgi?id=2419

**Review by corthon — APPROVED** [review]

**Pull request merged**